masterlist.yaml

```
@@ -3221,6 +3221,33 @@ plugins:
       - Hair
       - R.Relations
 
+  - name: '(Kart Hair.*|karthair.*)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/48588/'
+        name: 'Lore-Friendly Replacer for Default Hair for OCO v2'
+  - name: '(Kart Hair Replace v01|Kart Hair A Less Rude Mohawk)\.esp'
+    tag:
+      - Deactivate
+      - Graphics
+      - NoMerge
+  - name: 'Kart Hair Replace v01.esp'
+    clean:
+      - crc: 0x35CD9F85
+        util: 'TES4Edit v4.0.4b'
+  - name: 'Kart Hair A Less Rude Mohawk.esp'
+    clean:
+      - crc: 0x75B1E596
+        util: 'TES4Edit v4.0.4b'
+  - name: 'karthairdumpvanilla.esp'
+    tag: [ R.Hair ]
+    clean:
+      - crc: 0x23254AC8
+        util: 'TES4Edit v4.0.4b'
+  - name: 'karthairdump.esp'
+    clean:
+      - crc: 0xD90F3645
+        util: 'TES4Edit v4.0.4b'
```

These plugins should be sorted alphabetically (ignoring the extension):

```suggestion
  - name: 'karthairdump.esp'
    clean:
      - crc: 0xD90F3645
        util: 'TES4Edit v4.0.4b'
  - name: 'karthairdumpvanilla.esp'
    tag: [ R.Hair ]
    clean:
      - crc: 0x23254AC8
        util: 'TES4Edit v4.0.4b'
```